src/Api/Security/AdminAuthenticationProvider.php

```diff
@@ -110,12 +110,13 @@ public function clearCache(): void
 
     private function getMboToken(): string
     {
-        $employee = $this->context->employee;
-        if (!($employee instanceof \Employee)) {
-            throw new UnauthorizedException('User is not connected');
+        if($this->context && $this->context->employee && $this->context->employee instanceof \Employee) {
+            $salt = $this->context->employee->id;
+        } else {
+            $salt = 'APIClient';
```

Comment:
Do you prefer to keep a generic slate for all API clients? Because if you want to customize one then you could instead use the `ApiClientContext` service and get the ApiClient->id when it's present

If neither Employee nor ApiClient are present, then the `UnauthorizedException` would make sense